packages/env/lib/config.js

```diff
@@ -257,6 +257,21 @@ function parseSourceString( sourceString, { workDirectoryPath } ) {
 		};
 	}
 
+	const zipFields = sourceString.match(
+		/^https?:\/\/([^\s$.?#].[^\s]*)\.zip$/
```

Comment:
My first thought was that we should only capture the last part of the URL (`^https?:\/\/.+\/(.+)\.zip$`) so that the `basename` for e.g. `https://example.com/path/to/wordpress-beta.zip` is `wordpress-beta`. This matches what we're doing for the git source.

But, now that I think about it, this introduces the risk of collisions and I wonder if we should change how the git source works... 😕 

Any thoughts?

Reply:
Yeah, I think that ZIP file URLs are more likely to cause this issue, which is why I didn't change the git source yet. I don't think it will be a problem because we use Github, so it would most likely be a fork, and why would you clone the original repo and the fork in the same environment?